uitest/README.md

```diff
@@ -0,0 +1,43 @@
+## Dev Home UI Testing
+### How to run UI tests
+1. Open Dev Home solution in Visual Studio
+2. Install the version of Dev Home you want to test.
+   - Note: For testing a developer build of Dev Home, build and deploy the solution from Visual Studio.
+3. From the command prompt, start the Windows Application Driver service:
+```cmd
+"C:\Program Files\Windows Application Driver\WinAppDriver.exe"
+```
+4. From Visual Studio, navigate to the "Test Explorer" tab and locate the `DevHome.UITest` set of tests.
+5. Select a test and run it.
+    - Note: Once the test starts, avoid interacting with your machine (e.g. move mouse, use keyboard) to allow the test to navigate the Dev Home app and xecute the test.
+
+### How to configure the application settings for a Dev Home release
```

Comment:
When would I want to use/update/add to (?) these settings?

Reply:
App settings files are used for configuring a .Net application, in this case it's the UI tests. Any time a contributor wants to add a global variable to the entire set of tests, it can be done inside the JSON file. This approach allows us to easily configure the application from a single source without necessarily having to change the code and recompile the project. Furthermore, it gives us the option to run the same binaries for different flavors of Dev Home (canary, dev, preview/prod) where some global variables values are different.

Example of some fields we can define in the JSON:
- PackageFamilyName: Which Dev Home application to run (Canary, Dev, Release)
- Widget.IdPrefix: The prefix for a widget id that will be used for the widget tests. For example, we can test Dev Home (Dev) application with the Dev Home (Canary) widgets (may or may not necessarily be useful here, but we can do that 🙂)
- DefaultTimeOut: Default timeout value when attempting to perform a UI action with retry
- \<feature flag\>: A boolean to enable disable a feature flag
- Etc ...